### PR TITLE
Made clarifications about the zone ID

### PR DESCRIPTION
**Pull request by 123aswin123**
I dont have a domain on cloudflare, all I have is a workers.dev domain. This instruction regarding the zone ID on the CLI was confusing. Hence the edit.

### CODE REVIEW

File: src/settings/toml/manifest.rs
Comment (xtuc) [inline src/settings/toml/manifest.rs:343]:
```suggestion
                "You can find your {} in the right sidebar of your account's Workers page, and {} in the right sidebar of a zone's overview tab at {} (If you have only a workers.dev domain, you can skip adding the {} )",
```

Comment (xtuc) [inline src/settings/toml/manifest.rs:343]:
```suggestion
                "You can find your {} in the right sidebar of your account's Workers page, and {} in the right sidebar of a zone's overview tab at {} (if you have only a workers.dev domain, you can skip adding the {} )",
```

**Review by xtuc — APPROVED** [review]

**Review by nataliescottdavidson — APPROVED** [review]

**Comment by 123aswin123** [comment]
@nilslice can you merge this?

**Comment by nilslice** [comment]
@123aswin123 - would you be able to resolve the conflicts & rebase? If so, we can merge it and it can be in the upcoming release tomorrow.

**Comment by 123aswin123** [comment]
@nilslice done 👍

**Comment by nilslice** [comment]
Thanks! Once checks pass, I'll merge it in. Much appreciated!

**Comment by jyn514** [comment]
```
error[E0425]: cannot find value `account_id_msg` in this scope
   --> src/settings/toml/manifest.rs:492:17
    |
492 |                 account_id_msg, zone_id_msg, dash_url, zone_id_msg
    |                 ^^^^^^^^^^^^^^ help: a local variable with a similar name exists: `account_id_env`
```

**Comment by 123aswin123** [comment]
@nilslice The linter is complaining about a similar variable name, I am not the person that declared this variable.....

File: src/settings/toml/manifest.rs
Comment (jyn514) [inline src/settings/toml/manifest.rs:492]: You're using a variable that was removed sometime in the last 6 months. You could see how it was calculated before and do the same here.

Comment (123aswin123) [reply src/settings/toml/manifest.rs:492]: Makes sense....

Comment (123aswin123) [reply src/settings/toml/manifest.rs:494]: added variable, a maintainer needs to approve the workflow...

Comment (123aswin123) [reply src/settings/toml/manifest.rs:494]: https://github.com/cloudflare/wrangler/pull/1596/checks?check_run_id=3194423646 Do you have any idea as to why this is failing, there isn't much context.....

Comment (123aswin123) [reply src/settings/toml/manifest.rs:493]: I think it was the additional new line.... and I have removed it now.....

**Pull request merged**
